### PR TITLE
*: support select tidb_is_ddl_owner, to figure out whether tidb instance is ddl owner

### PR DESCRIPTION
**Pull request by winkyao**
After this PR, we can use `select tidb_is_ddl_owner()` to figure out whether the connected TiDB is DDL owner.
 
```
mysql> select tidb_is_ddl_owner();
+---------------------+
| tidb_is_ddl_owner() |
+---------------------+
|                   1 |
+---------------------+
1 row in set (0.00 sec)
```

### CODE REVIEW

File: expression/builtin_info.go [hunk 1]
```diff
@@ -346,6 +347,39 @@ func (b *builtinTiDBVersionSig) evalString(_ types.Row) (string, bool, error) {
...
+	newSig := &builtinTiDBIsDDLOwnerSig{}
+	newSig.cloneFrom(&b.baseBuiltinFunc)
+	return newSig
+}
+
+// evalInt evals a builtinTiDBIsDDLOwnerSig.
+func (b *builtinTiDBIsDDLOwnerSig) evalInt(_ types.Row) (res int64, isNull bool, err error) {
+	ddlOwnerChecker := b.ctx.DDLOwnerChecker()
+	if ddlOwnerChecker.IsOwner() {
+		res = 1
```
Comment (zimulala) [inline expression/builtin_info.go]: Could we replace `1` with `true`? I think it's more readable.

**Comment by shenli** [comment]
LGTM

File: expression/builtin_info.go (hunk 1 above)
Comment (winkyao) [reply expression/builtin_info.go]: 1 is enough

File: expression/builtin_info.go [hunk 2]
```diff
@@ -346,6 +347,39 @@ func (b *builtinTiDBVersionSig) evalString(_ types.Row) (string, bool, error) {
...
+
+func (b *builtinTiDBIsDDLOwnerSig) Clone() builtinFunc {
+	newSig := &builtinTiDBIsDDLOwnerSig{}
+	newSig.cloneFrom(&b.baseBuiltinFunc)
+	return newSig
+}
+
+// evalInt evals a builtinTiDBIsDDLOwnerSig.
+func (b *builtinTiDBIsDDLOwnerSig) evalInt(_ types.Row) (res int64, isNull bool, err error) {
+	ddlOwnerChecker := b.ctx.DDLOwnerChecker()
```
Comment (zimulala) [inline expression/builtin_info.go]: Could we replace `b.ctx.DDLOwnerChecker()` with `domain.GetDomain(b.ctx).DDL().OwnerManager()`? Then we can remove the field of `ddlOwnerChecker`.

Comment (winkyao) [reply expression/builtin_info.go]: NO. session cannot import ddl, it's a recycle import.

File: expression/builtin_info.go (hunk 1 above)
Comment (ciscoxll) [reply expression/builtin_info.go]: You can use `1`, but need to write the documentation

**Pull request merged**